kubernetes/kube.libsonnet

```diff
@@ -144,7 +144,7 @@
       spec: {
         group: group,
         names: names,
-        versions: error 'versions required',
+        version: version,
```

Comment:
Shouldn't this be `versions`, per [the API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#customresourcedefinitionspec-v1-apiextensions-k8s-io)?

Reply:
Yes, I had local changes, pushed now.